README.md

```diff
@@ -32,6 +32,18 @@ Visit the [Ably Asset Tracking](https://ably.com/documentation/asset-tracking) d
 - [Introducing Ably Asset Tracking - public beta now available](https://ably.com/blog/ably-asset-tracking-beta)
 - [Accurate Delivery Tracking with Navigation SDK + Ably Realtime Network](https://www.mapbox.com/blog/accurate-delivery-tracking)
 
+## Requirements
+
+- iOS 12.0+ / iPadOS 12.0+
+- Xcode 12.5+
```

Comment:
Minor: Does SDK really need Xcode 12.5? I'm asking because 12.5 is the first version that requires macOS Big Sur so I'd be careful defining it. Wouldnt just 12.X be enough? 

As a side note: Xcode 12.5 (and Big Sur in general) is not yet available in GitHub actions.

Reply:
Thanks for mentioning. I wasn't sure how low to go on that requirements. So I specified the version that I am using. I'm happy to change it to 12.x, but I cannot confirm it (my internet is too slow to download more versions of Xcode, they are 12GB each?).

Reply:
For sure we could lower it to 12.4 as it's used now in GH Actions. It'd also satisfy me because it's the last Xcode which does work on macOS Catalina, so we won't force anyone to upgrade OS.

Reply:
Cool, that's a good idea. Thanks